### PR TITLE
add support for Sonata 2019

### PR DESCRIPTION
**Pull request by AskAlice**
the tuning needs some refinement, but here is the interface in use:
https://my.comma.ai/cabana/?route=b2a38c712dcf90bd%7C2020-05-18--18-12-48&exp=1621380082&sig=rea0t%2Fdg9EyGvVWo7jcMKSd4VWU7Bw3ra6ykjlfuAVk%3D&max=5&url=https%3A%2F%2Fchffrprivate-vzn.azureedge.net%2Fchffrprivate3%2Fv2%2Fb2a38c712dcf90bd%2F5aeb4919fde73c8cd43de3dc3a0af44e_2020-05-18--18-12-48

### CODE REVIEW

**Comment by pd0wm** [comment]
Can you call it SONATA_2019?

**Comment by AskAlice** [comment]
@pd0wm done

File: README.md [hunk 1]
```diff
@@ -146,10 +146,12 @@ Community Maintained Cars and Features
 | Hyundai   | Kona 2019 EV<sup>2</sup>      | SCC + LKAS        | Stock            | 0mph               | 0mph         |
 | Hyundai   | Palisade 2020<sup>2</sup>     | All               | Stock            | 0mph               | 0mph         |
 | Hyundai   | Santa Fe 2019                 | All               | Stock            | 0mph               | 0mph         |
+| Hyundai   | Sonata 2019                   | All               | Stock            | 0mph               | 0mph         |
 | Hyundai   | Sonata 2020                   | All               | Stock            | 0mph               | 0mph         |
 | Jeep      | Grand Cherokee 2016-18        | Adaptive Cruise   | Stock            | 0mph               | 9mph         |
 | Jeep      | Grand Cherokee 2019           | Adaptive Cruise   | Stock            | 0mph               | 39mph        |
 | Kia       | Forte 2018<sup>2</sup>        | SCC + LKAS        | Stock            | 0mph               | 0mph         |
+| Kia       | Forte 2019<sup>2</sup>        | SCC + LKAS        | Stock            | 0mph               | 0mph         |
```
Comment (AskAlice) [inline README.md:154]: this matches the fingerprint for forte 2018 so no additional fingerprint needs to be added

Comment (pd0wm) [reply README.md:154]: You can just put `Sonata 2019-2020` and `Forte 2018-2019`

**Comment by pd0wm** [comment]
I saw on discord the tuning needs some work?

**Comment by AskAlice** [comment]
Yes, it can be iterated. The person who can test this has limited experience with a terminal and I find it might be faster to upstream, then have them use some fork or script to help with finding tuning values.

**Comment by pd0wm** [comment]
If you add the SONATA_2019 to this list: https://github.com/commaai/openpilot/blob/master/selfdrive/test/test_car_models.py#L367 the tests will pass.

**Comment by TheSiskoDax** [comment]
@AskAlice @pd0wm I am available to will help in anyway I can. Although, as AskAlice said, I am not a developer. However, I good with a computer and am a quick learner! :)

**Pull request merged**